app/views/marine/sections/_social_sharing.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="social--sharing">
+  <a 
+    href="https://twitter.com/intent/tweet/?text=<%= url_encode('Explore the World\'s Marine Protected Areas') %>&url=<%= encoded_page_url %>" 
+    title="Share Marine Protected Planet on Twitter" 
+    class="social--share social--twitter"
+    target="_blank">
+  </a>
+
+  <a 
+    href="https://facebook.com/sharer/sharer.php?u=<%= encoded_page_url %>" 
+    title="Share Marine Protected Planet on Facebook" 
+    class="social--share social--facebook"
+    target="_blank">
+  </a>
+
+  <a 
+    href="https://www.linkedin.com/shareArticle?mini=true&url=<%= encoded_page_url %>" 
+    title="Share Marine Protected Planet on LinkedIn" 
+    class="social--share social--linkedin"
+    target="_blank">
+  </a>
+  <a 
+    href="mailto:?subject=<%= url_encode('Marine Protected Planet website') %>&body=<%= encoded_page_url %>" 
+    title="Share Marine Protected Planet via Email" 
+    class="social--share social--email" 
+    target="_self">
+  </a>
```

Comment:
Since all of these links have the same structure, it might be a good idea to put those into a Rails helper. Let me know if you need help with that :)